packs/actions/timely-dodge.json

```diff
@@ -0,0 +1,74 @@
+{
+    "_id": "gohBPHf4dxi4PkBk",
+    "img": "systems/pf2e/icons/actions/Reaction.webp",
+    "name": "Timely Dodge",
+    "system": {
+        "actionType": {
+            "value": "reaction"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "defensive",
+        "description": {
+            "value": "<p><strong>Trigger</strong> You are targeted by a Strike</p><hr /><p><strong>Effect</strong> You dart aside, gaining a +1 circumstance bonus to AC, then Step after the Strike resolves.</p>"
```

Comment:
Feels like we should mention the scaling AC with the higher level versions

Reply:
Each effect other than the lesser one has a description override with the right AC. Should I mention the scaling AC in the base item anyway?

Reply:
Ah, that's fine then